### PR TITLE
If no offline specified. We should remove the tmp dir if it is existed.

### PR DESCRIPTION
**Pull request by yxwu25**

### CODE REVIEW

File: bin/vue-init [hunk 1]
```diff
@@ -62,9 +63,11 @@ var to = path.resolve(rawName || '.')
 var clone = program.clone || false
 
 var tmp = path.join(home, '.vue-templates', template.replace(/\//g, '-'))
-if (program.offline) {
+if (program.offline && exists(tmp)) {
   console.log(`> Use cached template at ${chalk.yellow(tildify(tmp))}`)
   template = tmp
+} else {
+  rm(tmp)
```
Comment (znck) [inline bin/vue-init]: IMO it should overwrite existing file after successful download.

Comment (yxwu25) [inline bin/vue-init]: Yes, that's best. Needs another dir to cache the downloads. Otherwise the git will throw an error.

Comment (znck) [inline bin/vue-init]: Native `/tmp` (😝  And whatever alternative for windows) directory can be used for that.

**Comment by yyx990803** [comment]
Thanks for the PR and sorry for letting it sit for so long. As of now, we are no longer adding new features of vue-cli 2.x, so I am closing this PR. Again, thanks for the contribution.

**Pull request closed without merging**